Azure-Samples/azure-search-openai-demo
# Using the option --storagekey for the prepdocs script might fail

When using the option --storagekey for the prepdocs script the value might contain `==` base64 padding at the end. This will fail to successfully login because the script just removes the `=` signs during the split action. Copied the version from the app/start.ps1 which is better suited here.

## Purpose

* make the prepdocs script work with the --storagekey parameter with the value `$env:AZURE_STORAGE_KEY` that contain base64 padding characters

## Does this introduce a breaking change?

```
[ ] Yes
[ X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ X] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
* Get the code
* Get the StorageKey from the Azure Storage and make sure it has bas64 padding at the end (`==`)
* Add the AZURE_STORAGE_KEY to the azd environment `azd env set AZURE_STORAGE_KEY <<value>>`
* add the --storagekey $env:AZURE_STORAGE_KEY parameter to the script invocation in the prepdocs.ps1 file
* run the script it will complain about invalid padding because the equal signs have been stripped of
* with this fix it will work correctly


## What to Check
Verify that the following are valid
* processdocs script works

## Other Information
